README.md

```diff
@@ -308,6 +308,46 @@ This should be verified by executing the Pelion E2E python test library tests.
 
 <span class="notes">**Note:** Future version will bring in also firmware update test as part of the minimum test set.</span>.
 
+## Contributing platform support
+
+The contribution of platform support to this repository is restricted to Arm Mbed Partners and Arm Engineering teams. If you’d like to add a custom or community-based platform, please fork this repository and add it into your own account.
+Expectations on contributions:
+
+-	No code changes in `main.cpp`.
+This is a minimal and generic application that’s expected to work on out of the box with all platforms listed in the documentation and [Pelion Quick-start](https://os.mbed.com/guides/connect-device-to-pelion/) guide
+
+- No changes to the hash of `mbed-os.lib ` or `mbed-cloud-client.lib` files.
+These are maintained and updated by the Arm team.
+
+-	No extra files or `.mbedignore` with removal of Mbed OS code.
```

Comment:
If such things would be needed, they imply eithe
- lack of configuration (in which case an issue should be raised in [Mbed OS](https://github.com/ARMmbed/mbed-os/issues/) repository) or
- Mbed OS release used by this repo is not update-to-date (in which case you can raise an issue for the `mbed-os.lib` to be updated by the maintainers of this repo).